### PR TITLE
Organize files by filter type

### PR DESCRIPTION
**Pull request by antoinebhs**

### CODE REVIEW

**Comment by antoinebhs** [comment]
To avoid code duplication, I had to create AbstractLineFilter and AbstractTransformerFilter.

File: src/main/java/org/gridsuite/filter/server/dto/criteriafilter/AbstractLineFilter.java [hunk 1]
```diff
@@ -4,39 +4,32 @@
...
+import org.springframework.util.CollectionUtils;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.SortedSet;
 
 /**
- * @author Jacques Borsenberger <jacques.borsenberger at rte-france.com>
+ * @author Antoine Bouhours <antoine.bouhours at rte-france.com>
```
Comment (thangqp) [inline src/main/java/org/gridsuite/filter/server/dto/criteriafilter/AbstractLineFilter.java:24]: If considering this abstract class as new => change Copyright (c) 2021 to Copyright (c) 2023

File: src/main/java/org/gridsuite/filter/server/dto/criteriafilter/LineFilter.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+@ToString(callSuper = true)
+@Schema(description = "Line Filters", allOf = AbstractLineFilter.class)
+public class LineFilter extends AbstractLineFilter {
+    public EquipmentType getEquipmentType() {
```
Comment (thangqp) [inline src/main/java/org/gridsuite/filter/server/dto/criteriafilter/LineFilter.java:25]: better with @Override

**Review by thangqp — COMMENTED** [review]
In my opinion,  using ''IdentifierListxxx" is some thing so technical with 'List', may be simplified by "Identifierxxx" or "Identifiersxxx" (avec 's')?

**Review by thangqp — APPROVED** [review]

**Pull request merged**